validation/src/test/scala/hmda/validation/rules/lar/validity/V338Spec.scala

```diff
@@ -0,0 +1,51 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V338Spec extends LarEditCheckSpec {
+  property("All applicants not meeting the criteria must pass") {
+    forAll(larGen) { lar =>
+      whenever(applicantNotOk(lar)) {
+        lar.mustPass
```

Comment:
The "Ok" naming is a bit confusing.  It's confusing in both the production code and the test code, but I think these two lines are the best example of what confused me.  When the applicant is not OK, the LAR must pass the validation?

Unfortunately I do not yet have a better name to suggest.  Something that means "the kind of LAR to which this rule even applies," but ideally with fewer words.  I wonder whether that concept already appears elsewhere in the code, and if so, what we called it there.  @schbetsy, do you know of any examples?